README.md

```diff
@@ -7,15 +7,15 @@
 [![Waffle.io - Columns and their card count](https://badge.waffle.io/bayesimpact/encompass.svg?columns=all)](http://waffle.io/bayesimpact/encompass)
 
 ## Introduction
-Encompass is an analytics and mapping tool by [Bayes Impact](http://bayesimpact.org) that enables policymakers, researchers, and consumer advocates to analyze how accessibility to social services varies across demographic groups. Inadequate and untimely access to health care services is a major barrier to health equity for disadvantaged communities. Existing tools used to map systems at this scale are prohibitively expensive, require significant amounts of manual data processing, and are too coarse in their analysis methods to accurately depict accessibility issues. We set out to build a solution that eliminates those barriers.
+Encompass is an analytics and mapping tool by [Beacon Labs](http://thebeaconlabs.org) that enables policymakers, researchers, and consumer advocates to analyze how accessibility to social services varies across demographic groups. Inadequate and untimely access to health care services is a major barrier to health equity for disadvantaged communities. Existing tools used to map systems at this scale are prohibitively expensive, require significant amounts of manual data processing, and are too coarse in their analysis methods to accurately depict accessibility issues. We set out to build a solution that eliminates those barriers.
 
 This is an open-source project. We invite researchers, developers, and the public to contribute to our project. See below for details.
 
-__[Launch Encompass](https://encompass.bayesimpact.org)__
+__[Launch Encompass](https://encompass.thebeaconlabs.org)__
 
 ![alt text](data/images/encompass_texas.png)
 
 ## How to contribute
-__Researchers__: We’d love to collaborate with any researchers who might find our tool useful! Please let us know what other applications or datasets you would like to analyze with Encompass. Send your inquiries to [encompass@bayesimpact.org](mailto:encmpass@bayesimpact.org).
+__Researchers__: We’d love to collaborate with any researchers who might find our tool useful! Please let us know what other applications or datasets you would like to analyze with Encompass. Send your inquiries to [encompass@thebeaconlabs.org](mailto:encmpass@thebeaconlabs.org).
```

Comment:
UHHHHHHHH `encmpss` --> `encompass` por favor

Reply:
Hopefully no one tried to use this link to email us...

Reply:
fixed!